deployments/kubernetes/chart/reloader/values.yaml

```diff
@@ -91,6 +92,7 @@ reloader:
     annotations: {}
     labels:
       provider: stakater
+      base: stakater/reloader
```

Comment:
isnt this supposed to be under image?

Reply:
Updated the base under image. Thanks for pointing out.